Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mozillamarketplace/centos-python27-mkt:0.5
```

Comment:
Iis there any advantage to building on top of the centos image as opposed to busybox? I guess all these layers (for centos/python) are there (cos we're using them elsewhere) - so size shouldn't be any issue.